REDACTED-params.yaml

```diff
@@ -88,7 +88,6 @@ wavefront:
   jaeger-app-name-prefix: dpfeffer-acme-fitness
 velero:
   bucket: pa-dpfeffer-mgmt-velero
-  region: us-east-2
```

Comment:
Wouldn't this prevent things from working with AWS S3?

Reply:
I've modified the script to pull the aws.region config value as the target for velero backup.

Reply:
Ah I see it's pulled from the aws.region now in the case of AWS